runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -492,7 +499,8 @@ fn action_deploy_contract_per_byte(ctx: &mut EstimatorContext) -> GasCost {
         ys.push(cost);
     }
 
-    let (_base, per_byte_cost) = GasCost::least_squares_method_gas_cost(&xs, &ys);
+    let (_base, per_byte_cost) =
+        GasCost::least_squares_method_gas_cost(&xs, &ys).expect("Returned negative parameters");
```

Comment:
Here, I think it might be better to flag the cost as uncertain, rather than panic, so that, if you run a bunch of estimates, you still can conclude the rest of them.

Reply:
Heh, yeah I'm already running such code to actually get the estimate :)

I thought I will do all the functional changes in a separate PR, including the 5/4 factor and a few other things. But I can include this change here if you want.

Reply:
no opinion on this!

Reply:
Changed this to use the new tolerance factor, which should mean we avoid all runtime panics